tuple/sqlx/tuple_sketch_int64_a_not_b.sqlx

```diff
@@ -21,6 +21,19 @@ config { hasOutput: true }
 
 CREATE OR REPLACE FUNCTION ${self()}(sketchA BYTES, sketchB BYTES)
 RETURNS BYTES
+OPTIONS (
+  description = '''Computes a sketch that represents the scalar set difference of sketchA and not sketchB.
+Cardinality estimation accuracy, plots, error tables, and p value are the same as the Theta Sketch.
+This function only applies to Tuple Sketches with an INT64 summary column. 
+  
+Param sketchA: the first sketch "A" as BYTES. This may not be NULL.
+Param sketchB: the second sketch "B" as BYTES. This may not be NULL.
+Param seed: assume default = 9001
```

Comment:
it seems to me that mentioning a non-existent parameter just like an existing might be confusing. perhaps better something like this "This function assumes default seed"

Reply:
It is important to state what the assumed value is.  So somewhere I would have to list the assumed parameters.  I figured this list of params was as good as any.  If you are still uncomfortable I could do this instead:
Param sketchA: the first sketch "A" as BYTES. This may not be NULL.
Param sketchB: the second sketch "B" as BYTES. This may not be NULL.
Assumed Default Param seed = 9001
...

Reply:
Fixed as shown.

Reply:
I've changed the wording to make it abundantly clear:
`Assumed Default Param seed: 9001 `